picasso/src/main/java/com/squareup/picasso/result/HTTPFailure.java

```diff
@@ -0,0 +1,28 @@
+package com.squareup.picasso.result;
+
+import com.squareup.picasso.Downloader.ResponseException;
+
+public class HttpFailure extends Failure {
```

Comment:
Facepalm. I had some issues with Git not recognizing the file name change. Thought I had it fixed.